test/jdk/java/lang/Math/AbsTests.java

```diff
@@ -143,4 +160,90 @@ private static int testLongAbs(LongUnaryOperator absFunc,
             return 0;
         }
     }
+
+    // --------------------------------------------------------------------
+
+    private static float testInRangeFloatAbs() {
+        int errors = 0;
+        float[][] testCases  = {
```

Comment:
For the particulars of the test vector for abs, another way to structure this would be a 1-D array of positive numbers where nested test loops used the test value and its negation as the input with the positive number as the expected value.

Reply:
Changed to 1-D array but can be either positive or negative.